src/esp/assets/PhysicsManager.cpp

```diff
@@ -109,15 +104,16 @@ bool PhysicsManager::initScene(
   } else {                                                  // GLB mesh data
     LOG(INFO) << "Initialize GLB scene";
   }
-  sceneSuccess = physObject->initializeScene(info, mass, meshGroup, *bWorld_);
+  sceneSuccess = physObject->initializeScene(info, mass, meshGroup, *bWorld_.get());
```

Comment:
`.get()` is redundant.  `*bWorld_` will dereference the underlying pointer.

Reply:
Thanks